### ISSUE TITLE
[bind-key]How to use obsidian.el in Emacs 29? Thank you very much.

### ISSUE THREAD

**Issue by suliveevil** [licht1stein/obsidian.el#36]
Emacs Version 29.0.50 (9.0)

obsidian.el: 20221102.1643

when install obsidian.el and put config to init.el then eval-buffer:
```emacs-lisp
;; obsidian.el
(require 'obsidian)
(obsidian-specify-path "~/Documents/GitHub/Chaos")
;; If you want a different directory of `obsidian-capture':
(setq obsidian-inbox-directory "Inbox")

;; Replace standard command with Obsidian.el's in obsidian vault:
(bind-key (kbd "C-c C-o") 'obsidian-follow-link-at-point 'obsidian-mode-map)

;; Use either `obsidian-insert-wikilink' or `obsidian-insert-link':
(bind-key (kbd "C-c C-l") 'obsidian-insert-wikilink 'obsidian-mode-map)

;; Following backlinks
(bind-key (kbd "C-c C-b") 'obsidian-backlink-jump 'obsidian-mode-map)


;; Optionally you can also bind `obsidian-jump' and `obsidian-capture'
;; replace "YOUR_BINDING" with the key of your choice:
; (bind-key (kbd "YOUR_BINDING") 'obsidian-jump)
; (bind-key (kbd "YOUR_BINDING") 'obsidian-capture)

;; Activate detectino of Obsidian vault
(global-obsidian-mode t)
```

Emacs notifies: symbol's function definition is void: bind-key

**Comment by licht1stein** [comment]
Hi @suliveevil 

Looks like `bind-key` is actually a library, and not part of Emacs 🤦‍♂️
I will change the example in the readme to use built-ins.

**Comment by licht1stein** [comment]
Please check if this snippet works:

```elisp
;; Define obsidian-mode bindings
(add-hook
 'obsidian-mode-hook
 (lambda ()
   ;; Replace standard command with Obsidian.el's in obsidian vault:
   (local-set-key (kbd "C-c C-o") 'obsidian-follow-link-at-point)

   ;; Use either `obsidian-insert-wikilink' or `obsidian-insert-link':
   (local-set-key (kbd "C-c C-l") 'obsidian-insert-wikilink)

   ;; Following backlinks
   (local-set-key (kbd "C-c C-b") 'obsidian-backlink-jump)))

;; Optionally you can also bind `obsidian-jump' and `obsidian-capture'
;; replace "YOUR_BINDING" with the key of your choice:
(global-set-key (kbd "YOUR_BINDING") 'obsidian-jump)
(global-set-key (kbd "YOUR_BINDING") 'obsidian-capture)
```

**Comment by licht1stein** [comment]
Fixed in the last release (1.1.8)

**Comment by suliveevil** [comment]
Wow this is great, it takes effects.

I have another problem using obsidian.el that it slows down Emacs when doing `obsidian-follow-link-at-point` and sometimes create new file while the file is already exists in Obsidian Vault.
